src/server.mjs

```diff
@@ -654,9 +657,10 @@ function returnErrorMiddleware(err, req, res, next) {
 app.use(returnErrorMiddleware);
 
 // Gets the chunk path from a region specified in a bedfile 
```

Comment:
This comment makes this look like it is a pure function that determines what path on the local disk some data for a BED file ought to live at.

Actually it is responsible for downloading that data if it doesn't exist. That's surprising and we should probably include in the comment something about how this function ensures that the chunk data is available on the local disk and returns a path to it.